From 26a9a16d1ed3a1cb41cb01f11e654fe8af2b2c28 Mon Sep 17 00:00:00 2001
From: max ulidtko <ulidtko@gmail.com>
Date: Thu, 19 Sep 2019 11:31:50 +0300
Subject: [PATCH] Fix gtk-doc 1.31+ build failure

Since this library doesn't define any gobjects, the generated
xml/object_index.sgml is always empty, and xml/tree_index.sgml is altogether
absent with gtk-doc 1.31+.

Fix a possible doc build error by not including the tree_index.sgml.

See GitHub issue kupferlauncher/keybinder#16.
---
 docs/keybinder-docs.sgml | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/docs/keybinder-docs.sgml b/docs/keybinder-docs.sgml
index 8e2ba0c..1b2b266 100644
--- a/docs/keybinder-docs.sgml
+++ b/docs/keybinder-docs.sgml
@@ -21,14 +21,12 @@
         <xi:include href="xml/keybinder.xml"/>

   </chapter>
-  <chapter id="object-tree">
-    <title>Object Hierarchy</title>
-     <xi:include href="xml/tree_index.sgml"/>
-  </chapter>
+
   <index id="api-index-full">
     <title>API Index</title>
     <xi:include href="xml/api-index-full.xml"><xi:fallback /></xi:include>
   </index>
+
   <index id="deprecated-api-index" role="deprecated">
     <title>Index of deprecated API</title>
     <xi:include href="xml/api-index-deprecated.xml"><xi:fallback /></xi:include>
